src/widgetastic/widget/base.py

```diff
@@ -826,7 +826,7 @@ def __get__(self, o, t):
                         break
             else:
                 # Parse the callable's args and inject the correct args
-                c_args, c_varargs, c_keywords, c_defaults = inspect.getargspec(condition)
+                c_args, c_varargs, c_keywords, c_defaults = inspect.getfullargspec(condition)
```

Comment:
Please read carefully python documentation. There is a difference in the output between getargspec and getfullargspec:

https://docs.python.org/3/library/inspect.html#inspect.getargspec
https://docs.python.org/3/library/inspect.html#inspect.getfullargspec

Reply:
yes just updated with it 
(i was facing an issue to run it locally!)
Thank you for having look at my PR!